helix-term/src/ui/mod.rs

```diff
@@ -9,6 +10,7 @@ mod text;
 
 pub use completion::Completion;
 pub use editor::EditorView;
+pub use info::Info;
```

Comment:
Tests fail since `mod info` doesn't exist

Reply:
Yeah, accidentally included that.